src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IIfStatement.cs

```diff
@@ -0,0 +1,554 @@
+        IArgument (Matching Parameter: value) (OperationKind.Argument)
+          ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: Result1)
+  IIfStatement (OperationKind.IfStatement)
+    Condition: IBinaryOperatorExpression (BinaryOperationKind.IntegerGreaterThan) (OperationKind.BinaryOperatorExpression, Type: System.Boolean)
+        Left: ILocalReferenceExpression: m (OperationKind.LocalReferenceExpression, Type: System.Int32)
+        Right: ILiteralExpression (Text: 10) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 10)
+    IBlockStatement (1 statements) (OperationKind.BlockStatement)
+      IExpressionStatement (OperationKind.ExpressionStatement)
+        IInvocationExpression (static void System.Console.WriteLine(System.String value)) (OperationKind.InvocationExpression, Type: System.Void)
+          IArgument (Matching Parameter: value) (OperationKind.Argument)
+            ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: Result2)
+    IBlockStatement (1 statements) (OperationKind.BlockStatement)
+      IExpressionStatement (OperationKind.ExpressionStatement)
+        IInvocationExpression (static void System.Console.WriteLine(System.String value)) (OperationKind.InvocationExpression, Type: System.Void)
+          IArgument (Matching Parameter: value) (OperationKind.Argument)
+            ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: Result3)
+";
+            VerifyOperationTreeForTest<IfStatementSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17601, "https://github.com/dotnet/roslyn/issues/17601")]
+        public void IIfstatementWithElseIfConditionOutVar()
+        {
+            string source = @"
+using System;
+
+class P
+{
+    private void M()
+    {
+        var s = """";
+        /*<bind>*/if (int.TryParse(s, out var i))
+           Console.WriteLine($""i={i}, s={s}"");
+        else
+          Console.WriteLine($""i={i}, s={s}"");/*</bind>*/ 
+        
+    }
+}
+";
+            string expectedOperationTree = @"
+IIfStatement (OperationKind.IfStatement)
+  Condition: IInvocationExpression (static System.Boolean System.Int32.TryParse(System.String s, out System.Int32 result)) (OperationKind.InvocationExpression, Type: System.Boolean)
+      IArgument (Matching Parameter: s) (OperationKind.Argument)
+        ILocalReferenceExpression: s (OperationKind.LocalReferenceExpression, Type: System.String)
+      IArgument (Matching Parameter: result) (OperationKind.Argument)
+        ILocalReferenceExpression: i (OperationKind.LocalReferenceExpression, Type: System.Int32)
+  IExpressionStatement (OperationKind.ExpressionStatement)
+    IInvocationExpression (static void System.Console.WriteLine(System.String value)) (OperationKind.InvocationExpression, Type: System.Void)
+      IArgument (Matching Parameter: value) (OperationKind.Argument)
+        IOperation:  (OperationKind.None)
```

Comment:
>IOperation:  (OperationKind.None) [](start = 8, length = 33)

It looks like string interpolations are not supported, please create an issue to track that, unless we already have one.